### ISSUE TITLE
`python3 setup.py test` fails

### ISSUE THREAD

**Issue by risner** [Chia-Network/clvm#92]
File "/usr/local/lib/python3.8/unittest/loader.py", line 205, in loadTestsFromName
    test = obj()
TypeError: run_program() missing 3 required positional arguments: 'program', 'args', and 'operator_lookup'

Are tests not wired/written for setup.py yet?

**Comment by richardkiss** [comment]
Correct, tests aren't wired for `setup.py`. Using `py.test tests` works for now. A PR to connect the two would be great.

**Comment by risner** [comment]
Thanks for the reply.

I’m afraid I’m more a perl/php/C programmer than python. I’ve 
spent endless hours trying to read python code and have made virtually 
no progress. So It’d take me weeks to be able to do so.

Risner

**Comment by poypoyan** [comment]
Hi! I guess the most "ad hoc" way to do this is to add `test_suite="tests"` in the setup (thanks SO), as well as adding the version as being said in issue  #93 . However, doing `setup.py test` is deprecated, see https://github.com/pypa/setuptools/issues/1684 .
